<launch>

	<param name="/SimSettings/PixelStdDev" type="int" value="10"/>
	<param name="/SimSettings/ImgHeight" type="int" value="768"/>
	<param name="/SimSettings/ImgWidth" type="int" value="1024"/>
	<param name="/SimSettings/PixelHeight" type="double" value="0.0000067"/>
	<param name="/SimSettings/PixelWidth" type="double" value="0.0000067"/>

	<param name="/LeftCam/Intrinsic/M11" value= "923.5295" type="double"/>
	<param name="/LeftCam/Intrinsic/M12" value= "0.0" type="double"/>
	<param name="/LeftCam/Intrinsic/M13" value= "507.2222" type="double"/>
	<param name="/LeftCam/Intrinsic/M21" value= "0.0" type="double"/>
	<param name="/LeftCam/Intrinsic/M22" value= "922.2418" type="double"/>
	<param name="/LeftCam/Intrinsic/M23" value= "383.5822" type="double"/>
	<param name="/LeftCam/Intrinsic/M31" value= "0.0" type="double"/>
	<param name="/LeftCam/Intrinsic/M32" value= "0.0" type="double"/>
	<param name="/LeftCam/Intrinsic/M33" value= "1.0" type="double"/>
	<param name="/LeftCam/Distortion/D1" value="0.0" type="double"/>
	<param name="/LeftCam/Distortion/D2" value="0.0" type="double"/>
	<param name="/LeftCam/Distortion/D3" value="0.0" type="double"/>
	<param name="/LeftCam/Distortion/D4" value="0.0" type="double"/>
	<param name="/leftCam/Distortion/D5" value="0.0" type="double"/>
	<param name="/LeftCam/Translation/X" value="-2.2005" type="double"/>
	<param name="/LeftCam/Translation/Y" value="0.0" type="double"/>
	<param name="/LeftCam/Translation/Z" value="0.0" type="double"/>
	<param name="/LeftCam/Rotation/Yaw" value="0.0" type="double"/>
	<param name="/LeftCam/Rotation/Pitch" value="0.0" type="double"/>
	<param name="/LeftCam/Rotation/Roll" value="0.0" type="double"/>
			

	<param name="/RightCam/Intrinsic/M11" value= "911.3657" type="double"/>
	<param name="/RightCam/Intrinsic/M12" value= "0.0" type="double"/>
	<param name="/RightCam/Intrinsic/M13" value= "519.3951" type="double"/>
	<param name="/RightCam/Intrinsic/M21" value= "0.0" type="double"/>
	<param name="/RightCam/Intrinsic/M22" value= "909.3910" type="double"/>
	<param name="/RightCam/Intrinsic/M23" value= "409.0285" type="double"/>
	<param name="/RightCam/Intrinsic/M31" value= "0.0" type="double"/>
	<param name="/RightCam/Intrinsic/M32" value= "0.0" type="double"/>
	<param name="/RightCam/Intrinsic/M33" value= "1.0" type="double"/>
	<param name="/RightCam/Distortion/D1" value="0.0" type="double"/>
	<param name="/RightCam/Distortion/D2" value="0.0" type="double"/>
	<param name="/RightCam/Distortion/D3" value="0.0" type="double"/>
	<param name="/RightCam/Distortion/D4" value="0.0" type="double"/>
	<param name="/RightCam/Distortion/D5" value="0.0" type="double"/>
	<param name="/RightCam/Translation/X" value="2.2000" type="double"/>
	<param name="/RightCam/Translation/Y" value="0.0" type="double"/>
	<param name="/RightCam/Translation/Z" value="0.0" type="double"/>
	<param name="/RightCam/Rotation/Yaw" value="10.0" type="double"/>
	<param name="/RightCam/Rotation/Pitch" value="0.0" type="double"/>
	<param name="/RightCam/Rotation/Roll" value="0.0" type="double"/>



	<param name="/Options/FileDir" value="/home/ryan/GitHub_WorkSpace/data/"/>
	<param name="/Options/FileName" value="defaultConfig.xml"/>
	<node pkg="shared_files" name="config_node" type="config_node" output="screen"/>
</launch>
